ct-aa/dotTheory---JavaScript
# ✨ JS-601: Write a Template Literal

# Related Tickets
- [JS-601](https://codeandtheory.atlassian.net/browse/JS-601)

# Description
Added the **Write a Template Literal** challenge solutions.

# Validation
* Open the **Write a Template Literal** folder.
* Open the app.js on the code editor.
* The value of variable **drink** should be converted to a template literal

# Risks
N/A
